ideascube/mediacenter/forms.py

```diff
@@ -17,6 +18,9 @@ class Meta:
         widgets = {
             'lang': LangSelect,
         }
+        labels = {
+            'original': _('Source file (maximum size : 500 Mo)'),
```

Comment:
Yes, it is better to have it in the help text.
I didn't change the verbose_name as this can be use elsewhere and the size limit make sense only in the upload form. (I don't know where it is used (and if), but it could)